tests/v2/validation/rbac/etcdbackup/etcd_backup_test.go

```diff
@@ -0,0 +1,116 @@
+//go:build (validation || infra.any || cluster.any || stress) && !sanity && !extended
+
+package etcd
+
+import (
+	"strings"
+	"testing"
+
+	rbac "github.com/rancher/shepherd/extensions/rbac"
+	"github.com/rancher/shepherd/clients/rancher"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	"github.com/rancher/shepherd/extensions/clusters"
+	"github.com/rancher/shepherd/extensions/etcdsnapshot"
+	"github.com/rancher/shepherd/extensions/projects"
+	"github.com/rancher/shepherd/extensions/users"
+	"github.com/rancher/shepherd/pkg/session"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type ETCDRbacBackupTestSuite struct {
+	suite.Suite
+	client  *rancher.Client
+	session *session.Session
+	cluster *management.Cluster
+}
+
+func (etcd *ETCDRbacBackupTestSuite) TearDownSuite() {
+	etcd.session.Cleanup()
+}
+
+func (etcd *ETCDRbacBackupTestSuite) SetupSuite() {
+	etcd.session = session.NewSession()
+
+	client, err := rancher.NewClient("", etcd.session)
+	require.NoError(etcd.T(), err)
+
+	etcd.client = client
+	clusterName := client.RancherConfig.ClusterName
+	require.NotEmptyf(etcd.T(), clusterName, "Cluster name to install should be set")
+	clusterID, err := clusters.GetClusterIDByName(etcd.client, clusterName)
+	require.NoError(etcd.T(), err, "Error getting cluster ID")
+	etcd.cluster, err = etcd.client.Management.Cluster.ByID(clusterID)
+	require.NoError(etcd.T(), err)
+}
+
+func (etcd *ETCDRbacBackupTestSuite) testEtcdSnapshotCluster(role, name string, user *management.User) {
+
+	etcd.T().Logf("Created user: %v", user.Username)
+	standardUserClient, err := etcd.client.AsUser(user)
+	require.NoError(etcd.T(), err)
+
+	adminProject, err := etcd.client.Management.Project.Create(projects.NewProjectConfig(etcd.cluster.ID))
+	require.NoError(etcd.T(), err)
+
+	if role == rbac.StandardUser.String() {
```

Comment:
We need to compare member with ```rbac.StandardUser.String()``` instead of ```role``` otherwise it will not meet the condition and skip this if statement block.